Projects/MakeMeLaugh_Server/Assets/Scripts/TransportServer.cs

```diff
@@ -96,16 +90,17 @@ void Update()
                     if (playerMessage.MessageType == MessageType.NEW_CLIENT_CONNECTION)
                     {
                         registerNewPlayer(playerMessage, m_Connections[i]);
-                        SendMessageToPlayer(playerMessage.PlayerUuid, MessageType.PLAYER_ANSWER_SUBMISSION, "hello from server");
+                        
+                        // example of how to send data to a specific player
+                        // SendMessageToPlayer(playerMessage.PlayerUuid, MessageType.PLAYER_ANSWER_SUBMISSION, "hello from server");

```

Comment:
How to send things to a specific client by their UUID; Feel free to modify `MessageType` to add more game events